configure

```diff
@@ -2187,7 +2177,7 @@ else
   fi
   if $with_spacetime; then
     inf "        spacetime profiling....... yes"
-    if test "$with_spacetime_call_counts" = "true"; then
+    if test "$enable_call_counts" = "true"; then
```

Comment:
`with_spacetime_call_counts` is not defined anywhere. It seems to me that here should be `enable_call_counts` instead.

Reply:
This should have been changed in #1421.